lib/bindings/HTTPBindings.js

```diff
@@ -156,24 +153,29 @@ function returnCommands(req, res, next) {
         }
 
         function cleanCommand(command) {
-            return apply(iotAgentLib.removeCommand,
-                device.service,
-                device.subservice,
-                device.id,
-                command.name);
+            return apply(iotAgentLib.removeCommand, device.service, device.subservice, device.id, command.name);
         }
 
         updates = commandList.map(createCommandUpdate);
         cleanCommands = commandList.map(cleanCommand);
 
         async.parallel(updates.concat(cleanCommands), function(error, results) {
             if (error) {
-                config.getLogger().error(context,
-                    'Error updating command status after delivering commands for device [%s]',
-                    device.id);
+                config
```

Comment:
Better to have `config.getLogger()` in the same line.

Reply:
Fixed 9bb55ee